content/java/concepts/calendar/terms/after/after.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.after'
+Description: 'Compare whether the current instance of Calendar occurs after the time represented by the specified Object.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+  - 'Time'
+  - 'Datetime'
```

Comment:
```suggestion
Tags:
  - 'Methods'
  - 'Time'
  - 'Date'
```
We need to restrict tags to what is on this list: https://github.com/Codecademy/docs/blob/main/documentation/tags.md